### DISCUSSION TITLE
「パターン一覧から任意パターンを選択・表示できる」スレ

### DISCUSSION THREAD

**Discussion by msntts** [msntts/PatternLanguageEditor#7] [General]
「ゴール：パターンリストの各パターンをクリックしたらパターン表示が切り替わること」です！✊
タスクは[こちら](https://github.com/users/msntts/projects/3/views/13)
各タスクには実装方針を記入しているので、不明点などあればここで話あいましょう
ブランチ名は```pattern_selection ```です。

**Comment by kinakomochigit** [comment]
作業できるのは明日以降になりそうなので、いったん現状をpushしました。
クリックした・のidをstateに保存するところまでできています。
selectorの作成の仕方が分からず止まっている所です。

**Reply by msntts** [reply]
直しました👌
前回同様、修正内容見てもらって、動かしてみて、問題なさそうだったらプルリクエスト投げてください💪

**Reply by kinakomochigit** [reply]
dispatchはComponentではなくContainerのほうで書くのですねφ(..)
